# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/udev/hotfix-no-tests.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- systemd-256.2/meson.build.vanilla	2024-07-11 22:13:13.524524610 +0200
+++ systemd-256.2/meson.build	2024-07-11 22:15:27.650524048 +0200
@@ -2691,7 +2691,6 @@
 ############################################################
 
 subdir('rules.d')
-subdir('test')
 
 #####################################################################
 
